sycl/doc/design/CompileTimeProperties.md

```diff
@@ -45,7 +45,7 @@ for declaring global variables.  One such example is the
 ```
 namespace sycl::ext::oneapi {
 
-template <typename T, typename PropertyListT = properties<>>
+template <typename T, typename PropertyListT = experimental::empty_property_list>
```

Comment:
I would prefer reducing the ambiguity of the name by making it `empty_properties_t` to both make it clear this is a type alias and to not make it sound like it is related to `sycl::property_list`. Tag @gmlueck @rolandschulz @Pennycook .

On another note, we should also fix this design doc so it doesn't mention experimental extensions in non-experimental namespaces.

Reply:
Yes, I also prefer `empty_properties_t`.

> On another note, we should also fix this design doc so it doesn't mention experimental extensions in non-experimental namespaces

Yes, good catch.